lib/ae_mdw/db/sync/stats.ex

```diff
@@ -51,6 +59,44 @@ defmodule AeMdw.Db.Sync.Stats do
         &(&1 - 1)
       )
 
+  @spec statistics_mutations(time(), type_counts()) :: StatisticsMutation.t() | nil
+  def statistics_mutations(time, type_counts) do
```

Comment:
may we have a unit test for this syncing function and rename it for the purpose of the transactions stats?

Reply:
Done